project-base/storefront/components/Blocks/Product/LastVisitedProducts/LastVisitedProducts.tsx

```diff
@@ -0,0 +1,29 @@
+import { Heading } from 'components/Basic/Heading/Heading';
+import { ProductsSlider } from 'components/Blocks/Product/ProductsSlider';
+import { GetLastVisitedProductCatalogNumbers } from 'helpers/products/LastVisitedProducts';
+import { Webline } from 'components/Layout/Webline/Webline';
+import { useTypedTranslationFunction } from 'hooks/typescript/useTypedTranslationFunction';
+import { GtmProductListNameType } from 'types/gtm/enums';
+import { useProductsByCatnumsApi } from 'graphql/generated';
+
+const LastVisitedProducts: FC = () => {
+    const lastVisitedProductCatalogNumbers = GetLastVisitedProductCatalogNumbers();
```

Comment:
Typo with capital letter...?

```suggestion
    const lastVisitedProductCatalogNumbers = getLastVisitedProductCatalogNumbers();
```

Reply:
Fixed by moving it to useLastVisited, bcs it cant be called like this.